.github/workflows/lint-docs.yaml

```diff
@@ -1,12 +1,8 @@
 name: Check Markdown links
 
 on:
-  push:
```

Comment:
this was running as post submit, I don't think anyone is paying attention to it though. Running in PRs is enough IMO.

Reply:
The original thought was because while prow makes sure to rebase before pushing the commit, github actions don't lint on the rebase, so we're running both